test/cases/adapters/postgresql/postgresql_adapter_test.rb

```diff
@@ -15,6 +15,15 @@ def setup
         @connection_handler = ActiveRecord::Base.connection_handler
       end
 
+      def teardown
+        # use connection without follower_reads
```

Comment:
Could mention telemetry in this comment too.

Reply:
Done